apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -431,7 +438,15 @@ void CSMWorld::MiscRefIdAdapter::setData (const RefIdColumn *column, RefIdData&
         InventoryRefIdAdapter<ESM::Miscellaneous>::setData (column, data, index, value);
 }
 
-CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns) : ActorColumns (actorColumns) {}
+CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns)
+: ActorColumns (actorColumns),
+  mFlags(std::map<const RefIdColumn *, unsigned int>()),
```

Comment:
I have the opposite habit personally. No-op initializers add redundancy, so when I want to change or replace that member variable I have to go hunting down all the constructors to fix them. This isn't a very big deal, just found it odd.